src/main/java/org/jabref/logic/xmp/XmpUtilWriter.java

```diff
@@ -82,6 +85,15 @@ public static void writeXmp(Path file, BibEntry entry,
         XmpUtilWriter.writeXmp(file, bibEntryList, database, xmpPreferences);
     }
 
+    /**
+     * Writes the information of the bib entry to the dublin core schema using
+     * a custom extractor.
+     *
+     * @param dcSchema
```

Comment:
Remove non-commented parameters - This should automatically be done in IntelliJ if you use autoformat (ctrl+alt+l)